.github/workflows/dump.yml

```diff
@@ -0,0 +1,39 @@
+name: Dump
+on:
+  workflow_dispatch:
+
+jobs:
+  dump:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.IAM_ROLE }}
+          role-session-name: GitHub_to_AWS_via_FederatedOIDC
+          aws-region: us-east-1
```

Comment:
Does not work for me.
I have the following log
https://github.com/metabase/shoppy/actions/runs/14593467722/job/40933785568?pr=118

```
Run aws-actions/configure-aws-credentials@v4
  
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
```

Reply:
It is setup to use OIDC. The IAM role has been adjusted so that it will work with the `pull_request`.

Reply:
Solved